test/unit/test_runner.py

```diff
@@ -82,6 +82,16 @@ def kaboom():
         Runner(config=rc, cancel_callback=kaboom).run()
 
 
+def test_verbose_event_created_time(rc):
+    rc.command = ['echo', 'helloworld']
+    runner = Runner(config=rc)
+    status, exitcode = runner.run()
+    assert status == 'successful'
+    assert exitcode == 0
+    for event in runner.events:
+        assert 'created' in event, event
```

Comment:
fails prior to patch. But I wouldn't object to getting rid of it, since it's covered by modifications to assertions in other tests below.